modules/imgcodecs/include/opencv2/imgcodecs.hpp

```diff
@@ -131,13 +131,7 @@ returns an empty matrix ( Mat::data==NULL ). Currently, the following file forma
  */
 CV_EXPORTS_W Mat imread( const String& filename, int flags = IMREAD_COLOR );
 
-/** @brief Loads and resizes down an image from a file.
-@anchor imread_reduced
-@param filename Name of file to be loaded.
-@param flags Flag that can take values of @ref cv::ImreadModes
-@param scale_denom
- */
```

Comment:
Documentation should be added into header files (remove documentation comments from `.cpp` file)
Also use `@overload` doxygen command.